modules/text/sentiment_analysis/senta_bilstm/resources/module.py

```diff
@@ -0,0 +1,255 @@
+# -*- coding:utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import ast
+import json
+import numpy as np
+import os
+import six
+
+import paddle.fluid as fluid
+from paddle.fluid.core import PaddleTensor, AnalysisConfig, create_paddle_predictor
+import paddlehub as hub
+from paddlehub.io.parser import txt_parser
+from paddlehub.module.module import runnable
+
+from senta_bilstm_python.net import bilstm_net
+from senta_bilstm_python.processor import load_vocab, preprocess, postprocess
+
+
+class DataFormatError(Exception):
+    def __init__(self, *args):
+        self.args = args
+
+
+class Senta_BiLSTM(hub.Module):
```

Comment:
How about use SentaBiLSTM, and SentaLSTM, SentaBoW, SentaCNN as standard naming style

Reply:
And is it possible to unifiy 4 Senta module into single one Senta, then use a new param="bilstm, cnn bow" to control its structure?

Reply:
1. As for class name, I've already updated.
2. As for code reuse, now the module v2 haven't adapted the situation like senta well. I will inherit the ModuleV2 class  by NLPModuleV2 Class next time to fix this.